Sage/carbon
# feat(pager): describe pager using design tokens, update tests - FE-4915

### Proposed behaviour

Describes pager components using design tokens.
`StyledPagerLinkStyles` is styled as a `Link` component.
Updates tests when the above changes are made.

### Current behaviour

Pager component use the theme styled-component property.
Currently `StyledPagerLinkStyles` is styled as a simple button.

### Checklist




#### QA


### Additional context

![image](https://user-images.githubusercontent.com/44143630/153163454-4899c3bd-82ba-4391-b8f5-fe7d4b212259.png)

### Testing instructions

Only test once PR has been merged: https://github.com/Sage/carbon/pull/4760
The correct appearance can be checked in the components: Pager and Flat Table - paginated.
Use devtools to check if component use css variables.
